.travis.yml

```diff
@@ -33,13 +35,15 @@ env:
     - CONDA_CHANNEL_PRIORITY=True
     - NUMPY_VERSION=stable
 
+
+
 matrix:
   fast_finish: true
   include:
     - os : linux
       env: NAME='minimal'
            PYTHON_VERSION=2.7
-           SETUP_CMD='--with-memleak'
+           MAIN_CMD='python ./testsuite/MDAnalysisTests/mda_nosetests analysis auxiliary coordinates core formats topology --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak && python ./testsuite/MDAnalysisTests/mda_nosetests utils --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50 --with-memleak'
```

Comment:
@kain88-de I am pretty sure that with `;` the command run sequentially and not in parallel. And as mentioned in my previous command, you take the risk of running only half of the tests if you use `&&`.